sinatra/lesson1/tutorial.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: Introduction to Sinatra
+---
+
+## What is Sinatra?
+
+Sinatra is a super simple framework for making websites using Ruby and HTML.
+It's easy to understand, and a great tool to start learning web development
+with!
+
+Before you continue, make sure you'll completed the [HTML tutorial][html-tut],
+[version control tutorial][vc-tut], and the [Ruby tutorial][ruby-tut].
+
+[html-tut]: http://codebar.github.io/tutorials/html/lesson1/tutorial.html
+[vc-tut]: http://codebar.github.io/tutorials/version-control/introduction/tutorial.html
+[ruby-tut]: http://codebar.github.io/tutorials/ruby/lesson1/tutorial.html
+
+## Installing Ruby
+
+Before you begin, you must have Ruby installed on your machine. There are a
+couple of different ways you can get ruby setup.
+
+- Windows installation using [Ruby installer][ruby-installer]
+- Mac & Linux installation using [rbenv][rbenv-install]
+
+[rbenv-install]: https://github.com/sstephenson/rbenv "rbenv"
+[ruby-installer]: http://rubyinstaller.org/
+
+Once you've got Ruby installed, you'll need to install Sinatra. Open the
+terminal / command prompt, and type the following:
+
+```
+gem install sinatra
+```
+
+If you see a message like this (and some other stuff), you're good to go
+
+```
+Successfully installed sinatra-1.4.6
+```
+
+
+## What is this gem thing, anyway?
+
+Often when programming we'll write sections of code that could be used in other
+projects, preventing us from having to solve the same problems over and over.
+These pieces of reusable code are called libraries, though in the Ruby world we
+typically call our libraries "gems", purely because it's a cute name.
+
+To save us the trouble of copy and pasting library code into our projects,
+applications were written that will go and download the libraries you ask for,
+and then make them available in your project. These applications are called
+"package managers", and our Ruby package manager is called "gem".
+
+So when you ran `gem install sinatra` before you were instructing the `gem`
+package manager to go and download the `sinatra` gem so you could use it in
+your project without any extra fuss. How convenient!
```

Comment:
Great explanation. Is there something like this in the Ruby tutorial? If not, we should move it there.